assets/js/amp-editor-blocks.js

```diff
@@ -267,48 +315,171 @@ var ampEditorBlocks = ( function() {
 	};
 
 	/**
-	 * Default setup for inspector controls.
+	 * Get AMP Layout select control.
 	 *
 	 * @param {Object} props Props.
-	 * @return {Object|Element|*|{$$typeof, type, key, ref, props, _owner}} Inspector Controls.
+	 * @return {Object} Element.
 	 */
-	component.setUpInspectorControls = function setUpInspectorControls( props ) {
+	component.getAmpLayoutControl = function getAmpLayoutControl( props ) {
 		var ampLayout = props.attributes.ampLayout,
-			ampNoLoading = props.attributes.ampNoLoading,
-			isSelected = props.isSelected,
-			name = props.name,
 			el = wp.element.createElement,
-			InspectorControls = wp.editor.InspectorControls,
 			SelectControl = wp.components.SelectControl,
-			ToggleControl = wp.components.ToggleControl,
-			PanelBody = wp.components.PanelBody,
+			name = props.name,
 			label = __( 'AMP Layout' );
 
 		if ( 'core/image' === name ) {
 			label = __( 'AMP Layout (modifies width/height)' );
 		}
 
+		return el( SelectControl, {
+			label: label,
+			value: ampLayout,
+			options: component.getLayoutOptions( name ),
+			onChange: function( value ) {
+				props.setAttributes( { ampLayout: value } );
+				if ( 'core/image' === props.name ) {
+					component.setImageBlockLayoutAttributes( props, value );
```

Comment:
Per above, this will need to be moved to an `onChange` handler.

Reply:
This one is already in `onChange` handler.